docs/ftldns/configfile.md

```diff
@@ -39,6 +39,7 @@ The default settings for FTL's rate-limiting are to permit no more than `1000` q
 It is important to note that rate-limiting is happening on a *per-client* basis. Other clients can continue to use FTL while rate-limited clients are short-circuited at the same time.
 
 For this setting, both numbers, the maximum number of queries within a given time, **and** the length of the time interval (seconds) have to be specified. For instance, if you want to set a rate limit of 1 query per hour, the option should look like `RATE_LIMIT=1/3600`.
+The time interval is relative to when FTL has finished starting (so start of the daemon + possible delay by DELAY_STARTUP)  then it will advance in steps of the rate-limiting interval. If a client reaches the maximum number of queries it will be blocked until **the end of the current interval**. This will be logged to `/var/log/pihole-FTL.log`, e.g. `Rate-limiting 10.0.1.39 for at least 44 seconds`. If the client continues to send queries while being blocked already and they number of queries during the blocking exceeds the limit the client will be blocked **until the end of the next interval**. Clients continues to send queries above the limit, it will be blocked continuously.
```

Comment:
```suggestion
The time interval is relative to when FTL has finished starting (start of the daemon + possible delay by DELAY_STARTUP) then it will advance in steps of the rate-limiting interval. If a client reaches the maximum number of queries it will be blocked until **the end of the current interval**. This will be logged to `/var/log/pihole-FTL.log`, e.g. `Rate-limiting 10.0.1.39 for at least 44 seconds`. If the client continues to send queries to will continuously blocked as long as the rate of queries exceeds the set rate (`pihole-FTL.log` will contain lines like `Still rate-limiting 10.0.1.39 as it made additional 5007 queries`). As soon as the client requests less than the set limit, it will be unblocked(`Ending rate-limitation of 10.0.1.39`).
```